src/analysis/inner_product_space/basic.lean

```diff
@@ -1613,137 +1625,110 @@ variables {ι : Type*} [dec_ι : decidable_eq ι] (𝕜)
 open_locale direct_sum
 
 /-- An indexed family of mutually-orthogonal subspaces of an inner product space `E`. -/
```

Comment:
Could you change that docstring to better explain the new version? Maybe mention the case of subspaces as an example, as well as add another example to motivate the rather complex type you now have for V.